gateway/tox.ini

```diff
@@ -21,7 +21,7 @@ deps = -rrequirements.txt
        -rrequirements-dev.txt
 commands =
   pip check
-  python manage.py test
+  python manage.py test {posargs}
```

Comment:
For what is this change? Do you know, @korgan00 ?

Reply:
It is to be able to run certain tests using the tox command args